website/src/pages/summarize/story.tsx

```diff
@@ -0,0 +1,107 @@
+import { Textarea } from "@chakra-ui/react";
+import { useRef, useState } from "react";
+import useSWRImmutable from "swr/immutable";
+
+import fetcher from "src/lib/fetcher";
+
+const SummarizeStory = () => {
+  // Use an array of tasks that record the sequence of steps until a task is
+  // deemed complete.
+  const [tasks, setTasks] = useState([]);
+
+  const inputRef = useRef<HTMLTextAreaElement>(null);
+
+  // Fetch the very fist task.  We can ignore everything except isLoading
+  // because the onSuccess handler will update `tasks` when ready.
+  const { isLoading } = useSWRImmutable("/api/new_task/summarize_story", fetcher, {
+    onSuccess: (data) => {
+      console.log(data);
+      setTasks([data]);
+    },
+  });
+
+  // Trigger a mutation that updates the current task.  We should probably
+  // signal somewhere that this interaction is being processed.
+  const submitResponse = (task: { id: string }) => {
+    const text = inputRef.current.value.trim();
+    // TODO: implement after we have the backend endpoint
```

Comment:
can you uncomment and implement the mutation code here (mostly copied from `website/src/pages/grading/grade-output.tsx`) with a new field in the trigger body (that i'll make work in a following PR): `update_type: "text_reply_to_post"`

I can make this work pretty quick.

Reply:
@fozziethebeat gladly, quick question: `post` in the sense of HTTP POST, or post as in reddit post?

Reply:
post in `text_reply_to_post` refers to a row in the database in the post table. The post table allows to store conversation trees (threads or completions of different users or model sampling runs). The interactions like ratings and rankings are store in a separate table called post_reaction (currently users can only submit a single reaction to a given post).